profiler/src/main/java/com/splunk/opentelemetry/logs/BatchingLogsProcessor.java

```diff
@@ -102,6 +104,7 @@ private void syncDoExport() {
     List<LogData> batchCopy = copyClearBatch();
     if (batchCopy != null) {
       logger.debug("Sync export of batch with size {}.", batchCopy.size());
+      // TODO: Look at responses from this call.
```

Comment:
is this TODO something you wish to keep?

Reply:
Yeah, I think we should be using those return values....but I didn't want to increase scope in this PR. 
I could create a tracking issue if you'd like.